scripts/exgfs_wave_prep.sh

```diff
@@ -50,15 +50,13 @@ Starting MWW3 PREPROCESSOR SCRIPT for ${WAV_MOD_TAG}
                           PREP for wave component of NCEP coupled system
                           Wave component identifier : ${WAV_MOD_TAG}
 
-Starting at : $(date)
+Starting at : $(PDY)
```

Comment:
This will fail.
The intent here is to log the start time of this script via `$(date)`
```suggestion
```

Reply:
Yeah I caught another similar change that I reverted. Will check this one, thanks!